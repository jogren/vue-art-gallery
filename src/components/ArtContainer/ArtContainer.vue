<template>
  <div class="img-container">
    <div v-bind:key="image.id" v-for="image in images">
      <Artwork v-bind:image="image"/>
    </div>
  </div>
</template>

<script>
import Artwork from '../Artwork/Artwork';
export default {
  name: "ArtContainer",
  props: ["images"],
  components: {
    Artwork
  }
  
}
</script>


<style scoped>
.img-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
}

</style>