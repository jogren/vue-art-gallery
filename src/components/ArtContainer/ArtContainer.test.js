import { mount } from '@vue/test-utils'
import ArtContainer from './ArtContainer'

describe('ArtContainer', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(ArtContainer)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders correctly', () => {
    const wrapper = mount(ArtContainer)
    expect(wrapper.element).toMatchSnapshot()
  })
})