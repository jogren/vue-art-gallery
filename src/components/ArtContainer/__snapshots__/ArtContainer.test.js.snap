// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtContainer renders correctly 1`] = `
<div
  class="img-container"
/>
`;
